composer/yamls/models/web/classify_mnist.yaml

```diff
@@ -0,0 +1,41 @@
+train_dataset:
```

Comment:
I dont think we need all these yamls, as it'll be difficult to main both web and non-web versions.

Reply:
Maybe just keep _one_ for reference.

Reply:
Okay, removed web/ directory, keeping the Imagenet example.